test/binCases/errors/child-compilation-error/test.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = function testAssertions(code, stdout, stderr) {
```

Comment:
ˋbinCases` will be removed on v4 branch. You should move this test elsewhere.

Reply:
@sokra had instructed me to add a test here:

> * add test case in `test/binCases/...`
> * fix `NoEmitOnErrorsPlugin`
>     * add `test/statsCases` for this one.

If this isn't the right place for it, where should it go?